app/components/SheetMusic/comments/Comment.jsx

```diff
@@ -0,0 +1,52 @@
+
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+
+let Comment = React.createClass({
```

Comment:
In that case, use `const` instead of `let`.

```js
const Comment = React.createClass({ ... });
```

I know the rest of the code base isn't like this, but apparently it's good practice to use `const` for variables that aren't going to be modified, and `let` for variables that are. There's an ESLint rule for this, but it would have involved changing a huge amount of code so I didn't really bother.